Week-13/index.js

```diff
@@ -0,0 +1,101 @@
+const express = require("express");
+const request = require("request");
+
+const app = express();
+
+//requesting data for a particular city
+app.get("/", (req, res) => {
+  let city = req.query.city;
+  var request = require("request");
+
+  request(
+    `http://api.weatherapi.com/v1/current.json?key=372f208a3f284a61b4670325222910&q=${city}&aqi=no`,
+    function (error, response, body) {
+      if (response.statusCode === 200) {
+        res.send(`The weather in your city ${city} is ${body}`);
+      }
+    }
+  );
+});
+//requesting data for multiple cities
+app.get("/", (req, res) => {
```

Comment:
this and the above api seem to be similar apis and are using same kind of code. Perhaps you can abstract the common functionality and write a single handler